content/maxwell3_fdem/emi_tutorial/induced_currents_body.rst

```diff
@@ -0,0 +1,94 @@
+.. _induced_currents_body:
+
+Induced currents in the body
+============================
+
+Consider a simple equivalent circuit as shown in :numref:`Concepts_3loops_only`.
+
+.. figure:: ./images/Concepts_3loops_only.png
+   :align: center
+   :scale: 50%
+   :name: Concepts_3loops_only
+
+   Conceptual diagram for 3-loops system.
+
+Through the Tx (Loop1) we flow time-varying currents, then it generates time varying magnetic flux, which will exite the body (Loop2). Let :math:`\Phi^{p}` be the primary magnetic flux from the Tx. The body treats like and R-L circuilt.
+
+.. figure:: ./images/RLcircuit.png
+   :align: right
+   :scale: 60%
+   :name: RLcircuit
+
+   Conceptual diagram of RL circuit.
+
+Electrical impedance of the RL circuit can be written as
+
+.. math::
+    Z(\omega) = R + \imath \omega L,
+
+where R and L indicate resistance and inductance, respecitvely.
```

Comment:
spelling "respectively" (in sublime - there is a descent spell check - see image below) 

<img width="1238" alt="monosnap 2016-04-12 20-18-21" src="https://cloud.githubusercontent.com/assets/6361812/14481878/de95fd82-00eb-11e6-9caa-0d8fa3d82def.png">